### ISSUE TITLE
The hide/show function is not such hommization

### ISSUE THREAD

**Issue by oiahoon** [Mortennn/Dozer#4]
1. like the icon, **hide** can be a hollow circle, while **show** can be a solid circle;
2. and the trigger, in my mind, using double click to toggle the show/hide make more sense;
3. while showing/hiding can we apply a slide animation to the icons?
4. most important, if we can drag the icons into Dozer icon to hide it, can be awesome!

**Comment by Mortennn** [comment]
I agree. I am currently rewriting the project due to too many bugs related to the current approach Dozer takes on hiding the menubar icons. I think it will take about 2 weeks, but then, I will surely focus on implementing some of these features😀

**Comment by blennsuan** [comment]
Hi. Love this app. I was wondering if there was a way to hide the menu bar icons as soon as Iv'e logged in? I mean without having to do the toggle. Thanks! :)

**Comment by Mortennn** [comment]
I have created a new issue for the feature😀

**Comment by blennsuan** [comment]
Hey,

Thought I’d point out that the icon disappears when I do my custom keyboard shortcut toggle. :)
